spec/rubocop/cop/performance/unneeded_sort_spec.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::UnneededSort do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense when first is called with sort' do
+    inspect_source('[1, 2, 3].sort.first')
```

Comment:
Please use the `expect_offense` and `expect_no_offenses` helpers.

```ruby
expect_offense(<<-RUBY.strip_indent)
  [1, 2, 3].sort.first
            ^^^^^^^^^^ Use `min` instead of `sort...first`.
RUBY
```